parallel_cluster/README.md

```diff
@@ -208,6 +210,30 @@ The following worklows are working examples you can run through to see AWS Paral
 
 [bcbio variant calling example](working_examples/bcbio/bcbio-nextgen.md)
 
+## Uploading data back to s3
+
+By default, parallel cluster does not have write access to s3 buckets.  
+A workaround is taking your short-term local SSO credentials and importing them into parallel cluster.  
+
+From your local computer run:
+```
+aws2-wrap --profile "${AWS_PROFILE}" --export | xclip  # pbcopy if on a mac.
```

Comment:
Not 100% sure if we want to introduce yet another tool to do this when:

1) I showed @AndrewPattison how to do it via the UMCCR AWS SSO apps portal (https://umccr.awsapps.com/start) and he was fine with it?
2) We already have [`yawsso`](https://github.com/victorskl/yawsso/) from @victorskl to attain precisely that too? :-S

Reply:
I was ok with it but not leaving the command line would be ideal

Reply:
oh wow! yet another aws sso! 😅   -- yap right yawsso export creds token too!

Reply:
The pcluster conda env uses aws 1.18 though, yawsso relies on aws v2 being installed. From a recent AWS thread, I doubt aws v2 will come to pip any time soon. Can we stick with `aws2-wrap` for now? The portal approach is difficult to describe in a readme without a series of screen shots.

```
(pcluster) $ yawsso -p "${AWS_PROFILE}"
Required AWS CLI v2. Found aws-cli/1.18.121 Python/3.8.5 Linux/4.19.84-microsoft-standard botocore/1.17.44
(pcluster) $ aws2-wrap --profile "${AWS_PROFILE}" --export
export AWS_ACCESS_KEY_ID="CD"
export AWS_SECRET_ACCESS_KEY="AB"
export AWS_SESSION_TOKEN="XTZ"
export AWS_REGION="ap-southeast-2"
```

Reply:
Would it make sense to uninstall that deprecated aws cli from pcluster and force-install AWS v2 through image builder? That would make more sense (to me) that introducing yet another AWS SSO tool that has already been "yet-anotherized" (by Victor) :-S

Reply:
This is the local conda env that runs the `pcluster create`, `pcluster list` commands, not the AMI on which the cluster is built on (which is aws2 btw).

Reply:
Simply put, until [this issue](https://github.com/aws/aws-cli/issues/4947) goes away, we're bound on aws v1 on this conda env. It does make it annoying to run `aws sso` whilst on that environment, so I have aliases set up for that.

Reply:
In my global env, The command `aws2` is soft-linked to `aws`. @victorskl could `yawsso` could first search for `aws2` and then fall back to `aws`? Similar to `python3` to `python` soft-links

Reply:
Nm, I can see that can be done with the -b flag

Reply:
`yawsso --profiles "${AWS_PROFILE}" --bin "$(which aws2)" --export-vars` is an alternative but requires users to have the `aws2 -> aws` softlink set up.